src/tests.rs

```diff
@@ -373,3 +373,45 @@ mod from_iterator {
 		);
 	}
 }
+
+// See <https://github.com/Robbepop/string-interner/issues/9>.
+mod clone_and_drop {
+	use super::*;
+
+	fn clone_and_drop() -> (DefaultStringInterner, Sym) {
+		let mut old = DefaultStringInterner::new();
+		let foo = old.get_or_intern("foo");
+
+		let new = old.clone();
+		let _bar = old.get_or_intern("bar");
```

Comment:
Is this needed? Please add a comment.

Reply:
At the time I wrote it, I was concerned about optimization, because `.clone()` seems completely unnecessary without `let _bar = old.get_or_intern(..);` and could be optimized away.

I confirmed that additional `.get_or_intern()` is unnecessary to reproduce the problem, so I'll remove it.